packages/cli/lib/services/dryrun.service.ts

```diff
@@ -14,6 +14,7 @@ import * as vm from 'node:vm';
 import * as url from 'url';
 import * as crypto from 'crypto';
 import * as zod from 'zod';
+import * as botbuilder from 'botbuilder';
```

Comment:
I might not have understood something, you are enabling this dep but not saying why we need botbuilder since it seems unused?

Reply:
I’m saying it’s needed in the runner but probably no one will use it for dry run so it might not be needed in the cli